pkg/generators/markers.go

```diff
@@ -164,6 +183,13 @@ func (c commentTags) Validate() error {
 		err = errors.Join(err, fmt.Errorf("invalid CEL tag at index %d: %w", i, celError))
 	}
 
+	if c.Format != "" {
+		_, ok := NameFormats[c.Format]
```

Comment:
We have accepted formats that are not name formats. See https://github.com/kubernetes/kubernetes/blob/69b0699610099f239296c2659a5f936a6573bf59/staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/validation/formats.go#L26-L51 for official list. These shouldnt desugar into CEL and should be emitted into the spec.Schema `Format` field